pkg/driver/node.go

```diff
@@ -103,7 +106,9 @@ func newNodeService(driverOptions *DriverOptions) nodeService {
 
 	// Remove taint from node to indicate driver startup success
 	// This is done at the last possible moment to prevent race conditions or false positive removals
-	go removeTaintInBackground(cloud.DefaultKubernetesAPIClient, removeNotReadyTaint)
+	time.AfterFunc(taintRemovalInitialDelay, func() {
```

Comment:
Any particular reason to add a 1 second delay? Wasn't mentioned in PR description or original bug

Reply:
Tested various intervals, first call expected failures observed at 900ms, none at 1 second.